rundeckapp/grails-spa/packages/ui-trellis/src/library/components/widgets/rundeck-info/RundeckInfo.spec.ts

```diff
@@ -120,6 +124,7 @@ describe("RundeckInfo", () => {
     expect(rundeckVersionComponent.exists()).toBe(true);
     expect(rundeckVersionComponent.props("title")).toEqual(latestProps.title);
     expect(rundeckVersionComponent.props("number")).toEqual(latestProps.full);
+    expect(rundeckVersionComponent.props("tag")).toEqual(latestProps.tag);
```

Comment:
included tag now here and also added one more expect for checking that the span of rundeckVersion has the expected text which is in previous commit. Please suggest If I am missing something.